scripts/build_models.sh

```diff
@@ -94,9 +96,9 @@ cd ..
 mkdir "$output" 
 mv "$tmpoutput"/bin/* "$output"
 
-rm -rf "$tmpoutput"
+#rm -rf "$tmpoutput"
```

Comment:
Enable the command? Temporary output is probably not necessary under normal circumstances...

Reply:
Enabled, It was only there while I was debugging it, and I forgot to uncomment it afterward.